packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -1132,6 +1162,38 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
         }
     }
 
+    protected async isDisallowedBasedOnRobotsFile(url: string): Promise<boolean> {
```

Comment:
my pet peeve: boolean methods with negated names :) I'd prefer `isAllowedByRobotsTxt` or sth, IMO both the implementation and usage might get a bit more intuitive.

This is a nit though.

Reply:
Yeah, same here, but I also really dislike the need to wrap async calls in parens to be able to negate them 😄 And since its a protected method, I went with the negated form to get around that part.

Reply:
Renamed to the positive form in the end.